diff --git a/src/Makefile b/src/Makefile
index cf3fd4d..91cdf6c 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -12,10 +12,10 @@ INCLUDE_DIRS :=../include ./
 INCLUDE_FILES := $(wildcard ../include/*.hrl) $(wildcard ./*.hrl) 
 
 EBIN_DIRS := $(wildcard ../libs/*/ebin)
-ERLC_FLAGS := +warnings_as_errors -W $(addprefix -I , $(INCLUDE_DIRS)) $(addprefix -pa , $(EBIN_DIRS))
+ERLC_FLAGS := +debug_info +warnings_as_errors -W $(addprefix -I , $(INCLUDE_DIRS)) $(addprefix -pa , $(EBIN_DIRS))
 
 ifdef TEST
-  ERLC_FLAGS += +debug_info -DTEST 
+  ERLC_FLAGS += -DTEST 
 endif
 
 ifdef NOLOG
